index.md

```diff
@@ -101,6 +101,10 @@ Selskapet har pensjonssparing hos Gjensidige for alle ansatte. Vi innbetaler 5%
 
 Det er åpnet opp for at de ansatte kan samle sine (tidligere) pensjonsavtaler hos Gjensidige. Hver ansatt må kontakte Gjensidige om dette.
 
+### Foreldrepermisjon
+
+Det å bli forelder er en fantastisk opplevelse og vi unner alle å ta en velfortjent foreldrepermisjon. Vi betaler full lønn i foreldrepermisjon og tiden du er i permisjon vil ikke gå utover din ansiennitet i selskapet. For at 321 skal betale mellomlegget mellom utbetaling fra Nav og full lønn må du ha jobbet i 321 i minimum ett år før du går ut i foreldrepermisjon.
```

Comment:
Slik det står nå med "Vi betaler full lønn" er det litt selvmotsigende. Hva med: "Det å bli forelder er en fantastisk opplevelse og vi unner alle å ta en velfortjent foreldrepermisjon. Tiden du er i permisjon vil ikke gå utover din ansiennitet i selskapet. For at 321 skal betale mellomlegget mellom utbetaling fra Nav og full lønn må du ha jobbet i 321 i minimum ett år før du går ut i foreldrepermisjon."

Reply:
Ja, ditt forslag er bra @ollis